rcl_lifecycle/src/rcl_lifecycle.c

```diff
@@ -0,0 +1,176 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#if __cplusplus
+extern "C"
+{
+#endif
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "rcl/rcl.h"
+#include "rcl/error_handling.h"
+
+#include "rcl_lifecycle/rcl_lifecycle.h"
+#include "rcl_lifecycle/transition_map.h"
+
+#include "com_interface.h"  // NOLINT
+#include "default_state_machine.h"  // NOLINT
+#include "states.h"  // NOLINT
```

Comment:
Here as well (and anywhere else you can think of that you could do this relative include).